demo-dep-output.yaml

```diff
@@ -1629,6 +1629,2681 @@
 - fileURI: file:///analyzer-lsp/examples/inclusion-tests/pom.xml
   provider: java
   dependencies: []
+- fileURI: file:///analyzer-lsp/examples/java-project/pom.xml
```

Comment:
We can find these now because we extract the pom.xml from the jar and place it in the correct location. I think this is better the previous approaches